scripts/operator/package-operator.sh

```diff
@@ -43,6 +44,10 @@ sed -i.bak "s|SNYK_OPERATOR_IMAGE_TAG_OVERRIDE|${NEW_OPERATOR_IMAGE_TAG}|g" "${T
 sed -i.bak "s|SNYK_MONITOR_IMAGE_TAG_OVERRIDE|${NEW_MONITOR_IMAGE_TAG}|g" "${TARGET_CSV}"
 rm "${TARGET_CSV}.bak"
 
+if [[ "$OPERATOR_REPLACES_VERSION" != "" ]]; then
+  echo "  replaces: snyk-operator.v${OPERATOR_REPLACES_VERSION}" >> "${TARGET_CSV}"
```

Comment:
just to make sure the the spaces at the beginning are intentional

Reply:
They are, since this is a YAML file 😄